src/mlpack/methods/ann/loss_functions/l1_loss.hpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file l1_loss.hpp
+ * @author Himanshu Pathak
+ *
+ * Definition of the L1 Loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTION_L1_LOSS_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTION_L1_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The L1 loss is a loss function that measures the mean absolute error (MAE) 
+ * between each element in the input x and target y 
+ *
+ * @tparam InputDataType Type of the input data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ */
+template <
+    typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat
+>
+class L1LOSS
```

Comment:
Do you mind to change the name to `L1Loss` to be more consistent with the rest of the codebase.

Reply:
Thanks for reviewing @zoq . I changed the name of the class. Builds are failing because sever is down